pkg/operator/ceph/csi/spec.go

```diff
@@ -105,6 +107,7 @@ var (
 	DefaultSnapshotterImage       = "k8s.gcr.io/sig-storage/csi-snapshotter:v4.2.0"
 	DefaultResizerImage           = "k8s.gcr.io/sig-storage/csi-resizer:v1.3.0"
 	DefaultVolumeReplicationImage = "quay.io/csiaddons/volumereplication-operator:v0.1.0"
+	DefaultCSIAddonsImage         = "quay.io/csiaddons/k8s-sidecar:latest"
```

Comment:
Is there a versioned tag besides `latest`? Similar to how we use version tags for the other images, it gives us a more stable deployment. `latest` may cause unexpected breaking changes.

Reply:
No, we don't have any specific tags for now. @nixpanic Correct me if I am wrong

Reply:
It is fragile for us to use the `latest` image. I would consider it a requirement that the csiaddons project start releasing versioned images before consuming them in Rook.

Reply:
`quay.io/csiaddons/k8s-sidecar:v0.1.0` is getting build at the moment, and should land within the next hour at https://quay.io/repository/csiaddons/k8s-sidecar?tab=tags

Eventhough this is a v0.1.0 release, we do not expect to need any changes to the container image in the near feature. The kubernetes-csi-addons project will do more releases, but those are targeting the `csiaddons/k8s-controller` and `csiaddons/k8s-bundle` container images.